mr/src/main/java/org/apache/iceberg/mr/hive/serde/objectinspector/IcebergObjectInspector.java

```diff
@@ -72,7 +85,7 @@ public ObjectInspector primitive(Type.PrimitiveType primitiveType) {
         primitiveTypeInfo = TypeInfoFactory.booleanTypeInfo;
         break;
       case DATE:
-        return IcebergDateObjectInspector.get();
+        return DATE_INSPECTOR.invoke();
```

Comment:
Minor: we could invoke this just once and store it in a `private static final` field. Same with timestamps. We could store both in fields and choose one using `adjustToUtc`. Not a big deal to always call the reflection method here, though.

Reply:
saved them into `private static final` fields